src/basic/points.js

```diff
@@ -138,7 +138,7 @@ Point.getDraw = function(c3ss, id) {
 
     point.collide = !getCollide(c3ss);
 	}
-
+  point.order = 0;
```

Comment:
This parameter is ignored when comp-op equals to overlay so it's presence makes no difference and the logic looks simpler to me.